muffnn/__init__.py

```diff
@@ -1,5 +1,6 @@
-from muffnn.base import MLPBaseEstimator
-from muffnn.mlp_classifier import MLPClassifier
-from muffnn.mlp_regressor import MLPRegressor
+from .mlp.mlp_classifier import MLPClassifier
+from .mlp.mlp_regressor import MLPRegressor
+from .autoencoder.autoencoder import Autoencoder
+from .core import TFPicklingBase
 
-__all__ = ['MLPBaseEstimator', 'MLPClassifier', 'MLPRegressor']
```

Comment:
So after python 3.2, removing this file is OK I think. From the setuptools docs

"Anyway, find_packages() walks the target directory, filtering by inclusion patterns, and finds Python packages (any directory). On Python 3.2 and earlier, packages are only recognized if they include an __init__.py file. Finally, exclusion patterns are applied to remove matching packages."

I think we might want to retain it for all python 3 versions?